flash

```diff
@@ -213,7 +213,7 @@ while [[ -z $PREVIEWER_PARAMTERS ]]; do
   case $PREVIEWER in
     bat) PREVIEWER_PARAMTERS="--theme='Solarized (dark)' --style=numbers --color=always" ;;
     cat) PREVIEWER_PARAMTERS="--number" ;;
-    *)   echo -e "$PREVIEWER is not a valid previewer. Use 'bat' or 'cat'. Falling back on default..." && read -s -p 'Press [Enter] to continue...' && PREVIEWER='bat' ;;
+    *)   echo -e "$PREVIEWER is not a valid previewer. Use 'bat' or 'cat'. Falling back on default..." && read -r -s -p 'Press [Enter] to continue...' && PREVIEWER='bat' ;;
```

Comment:
last little change for aesthetics and my own preference

add a line break to the echo'd text `default...\n`